src/common/legacy_config_opts.h

```diff
@@ -153,6 +153,7 @@ OPTION(ms_async_rdma_receive_queue_len, OPT_U32)
 OPTION(ms_async_rdma_support_srq, OPT_BOOL)
 OPTION(ms_async_rdma_port_num, OPT_U32)
 OPTION(ms_async_rdma_polling_us, OPT_U32)
+OPTION(ms_async_rdma_gid_idx, OPT_INT)
```

Comment:
if this option is not read on critical path, probably we don't need to add it to `legacy_config_opts.h`.

Reply:
I'll remove it from legacy_config_opts.h since rdma is not on critical path.